BUILD.gn

```diff
@@ -43,7 +43,7 @@ config("vulkan_internal_config") {
     cflags = [
       "-Wno-conversion",
       "-Wno-extra-semi",
-      "-Wno-implicit-fallthrough",
```

Comment:
I think you can just remove this line altogether since this array contains all of the "-Wno" exceptions

Reply:
Done!